.github/workflows/test-python-package.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Test Python Package
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9, "3.10"]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install flake8 pytest pytest-cov
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        if [ -f requirements-test.txt ]; then pip install -r requirements-test.txt; fi
+    - name: Lint with flake8
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 --statistics --count --exit-zero
```

Comment:
does this error out on the current main?

Reply:
This adds test to run remotely that didn't exist. Now PRs will have tests to validate working code.

Reply:
I don't believe so. I ran a test on a personal repo too to validate the flake8 would pass:
https://github.com/JGSweets/synthetic-data/actions/runs/3101760273/jobs/5023430869
However, tests won't pass until https://github.com/capitalone/synthetic-data/pull/272 bc of the seed issue.

Reply:
Ok I think merging this in first is good so we have automated tests for the other pr even if they fail currently for main